mock-tee-primitives/src/identity.rs

```diff
@@ -0,0 +1,163 @@
+// Copyright 2020-2022 Litentry Technologies GmbH.
+// This file is part of Litentry.
+//
+// Litentry is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Litentry is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
+
+#[cfg(all(not(feature = "std"), feature = "sgx"))]
+extern crate sgx_tstd as std;
+
+#[cfg(any(feature = "std", feature = "sgx"))]
+use std::format;
+
+#[cfg(all(not(feature = "sgx"), feature = "std"))]
+use serde::{Deserialize, Serialize};
+#[cfg(any(feature = "std", feature = "sgx"))]
+use sp_core::hexdisplay::HexDisplay;
+#[cfg(any(feature = "std", feature = "sgx"))]
+use std::vec::Vec;
+
+use codec::{Decode, Encode, MaxEncodedLen};
+use scale_info::TypeInfo;
+use sp_runtime::{traits::ConstU32, BoundedVec};
+
+pub type MaxStringLength = ConstU32<64>;
+pub type IdentityString = BoundedVec<u8, MaxStringLength>;
+
+#[derive(Encode, Decode, Clone, Debug, PartialEq, Eq, TypeInfo, MaxEncodedLen)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub enum SubstrateNetwork {
+	Polkadot,
+	Kusama,
+	Litentry,
+	Litmus,
+}
+
+#[derive(Encode, Decode, Clone, Debug, PartialEq, Eq, TypeInfo, MaxEncodedLen)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub enum EvmNetwork {
+	Ethereum,
+	BSC,
+}
+
+#[derive(Encode, Decode, Clone, Debug, PartialEq, Eq, TypeInfo, MaxEncodedLen)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub enum Web3Network {
+	Substrate(SubstrateNetwork),
+	Evm(EvmNetwork),
+}
+
+#[derive(Encode, Decode, Clone, Debug, PartialEq, Eq, TypeInfo, MaxEncodedLen)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub enum Web2Network {
+	Twitter,
+	Discord,
+	Github,
```

Comment:
@Kailai-Wang Are you using brand names here or just the name? 
Correct brand name for "Github" is "GitHub".

Reply:
No, it is just me trying to use the correct name, thanks!